src/seedu/addressbook/ui/Formatter.java

```diff
@@ -0,0 +1,33 @@
+package seedu.addressbook.ui;
+
+import java.util.List;
+
+public class Formatter {
```

Comment:
Missing header comment. All non trivial classes/methods should have javadoc format header comments.